packages/plugins/plugin-assistant/src/hooks/useChatServices.ts

```diff
@@ -43,49 +43,25 @@ export type UseChatServicesProps = {
  * Construct service layer.
  */
 export const useChatServices = ({ space }: UseChatServicesProps): Layer.Layer<ChatServices> | undefined => {
-  const toolRegistry = useToolRegistry();
-  // TODO(dmaretskyi): We can provide the plugin registry as a layer and then build the entire layer stack from there. We need to think how plugin reactivity affect our layer structure.
-  const toolResolver = useToolResolver();
-  const toolExecutionService = useToolExecutionService();
+  const aiServiceLayer =
+    useCapabilities(AssistantCapabilities.AiServiceLayer).at(0) ?? Layer.die('AiService not found');
```

Comment:
is this to handle the capability being missing than it throwing here? if so, TODO we should factor out a util for this

Reply:
This will make it throw inside chat-processor. Agree about factoring out, but I'm holding off on this until a pattern emerges.